datasource/management/commands/groot-352.py

```diff
@@ -0,0 +1,109 @@
+import re
+from datetime import datetime
+
+from django.core.exceptions import ValidationError
+from django.core.management.base import BaseCommand
+from django.core.validators import URLValidator
+from django.utils import timezone
+
+from brand.models import *
+from datasource.models import *
+
+
+class Command(BaseCommand):
+    help = "This script enters information from usnic into brands for FCU entity type"
+
+    def handle(self, *args, **kwargs):
+        print("Running db_changes_fcu.py")
+        rssd_ids = []
+        with open(r"datasource\management\commands\cdfi_rssd_numbers.txt", "r") as file:
+            rssd_ids = [line.strip() for line in file]
+        filtered_usnic = Usnic.objects.filter(entity_type="FCU").values(
+            "ncua",
+            "website",
+            "country",
+            "legal_name",
+            "rssd",
+            "lei",
+            "fdic_cert",
+            "ein",
+            "occ",
+            "thrift_hc",
+            "cusip",
+        )
+        # selects the two fields to check for duplicates
+        brand_values = Brand.objects.values("tag", "name")
+        existing_tags = [x["tag"] for x in brand_values]
+        existing_names = [x["name"] for x in brand_values]
+        for row in filtered_usnic:
+            if row["legal_name"] in existing_names:
+                continue
```

Comment:
Adding a continue statement will break out of the current interation of the loop and move onto the next. Given that, you don't need the else statement. Generally, the fewer levels of indentation you can use the better

Reply:
It would be nicer to just filter the list ahead of time though. In that case, cycle through `filtered_usnic` and create a new list of `filtered_usnic_to_add`

Then you can cycle over that again. I think that would help separate the logic

Reply:
```
usnic_to_insert = []
for row in filtered_usnic:
    name = row["legal_name"]
    if name not in existing_names:
        existing_names.append(name)
        usnic_to_insert.append(row)

for usnic in usnic_to_insert:
    ....
```